release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -81,6 +81,12 @@ For more information, see xref:../installing/installing-troubleshooting.adoc#ins
 [id="ocp-4-12-networking"]
 === Networking
 
+[id="ocp-4-12-k8s-nmstate-support-for-vsphere"]
+==== Kubernetes NMState in VMWare vSphere now supported
+Beginning with OpenShift Container Platform 4.12, you can configure the networking settings such as DNS servers/search domains, VLANs, bridges, and interface bonding using the Kubernetes NMState Operator on your VMware vSphere instance.
```

Comment:
```suggestion
Beginning with {product-title} 4.12, you can configure the networking settings such as DNS servers or search domains, VLANs, bridges, and interface bonding using the Kubernetes NMState Operator on your VMware vSphere instance.
```
See the ISG: https://www.ibm.com/docs/en/ibm-style?topic=punctuation-slashes

Reply:
Fixed these, thank you1